include/envoy/request_info/request_info.h

```diff
@@ -68,66 +68,125 @@ class RequestInfo {
    */
   virtual void onUpstreamHostSelected(Upstream::HostDescriptionConstSharedPtr host) PURE;
 
+  /**
+   * @return the # of body bytes received in the request.
+   */
+  virtual uint64_t bytesReceived() const PURE;
+
+  /**
+   * @return the protocol of the request.
+   */
+  virtual const Optional<Http::Protocol>& protocol() const PURE;
+
+  /**
+   * Set the request's protocol.
+   */
+  virtual void protocol(Http::Protocol protocol) PURE;
```

Comment:
I thought we mostly name things like this setProtocol but grepping for Set it looks like we're about 50/50.  I think setFoo is more clear than foo but given we're not consistent please consider this an optional suggestion :-)

Reply:
The existing methods on RequestInfo were overloaded, so I was being consistent (at least locally).

Reply:
Consistency is the best argument :-)  Let's leave it as-is then.